### DISCUSSION TITLE
Dependencies pada Capstone DA

### DISCUSSION THREAD

**Discussion by vidric** [teamalgoritma/community#650] [Q&A] [answered]
Apakah kita boleh menambahkan dependencies lain pada project capstone kita ? misal import datetime

**Comment by idoali** [comment, marked as answer]
Jika diperlukan bisa di install sendiri dengan cara yang sudah pernah diajarkan pak. Tetapi di `requirements.txt` sendiri biasanya sudah mengandung semua dependencies yang dibutuhkan.
